### PR TITLE
Porting docs, examples, tutorials and galleries

### PR DESCRIPTION
**Pull request by datumbox**
Related to #4679

### CODE REVIEW

File: torchvision/transforms/_presets.py [hunk 1]
```diff
@@ -106,18 +106,26 @@ def forward(self, img: Tensor, target: Optional[Tensor] = None) -> Tuple[Tensor,
 
 
 class OpticalFlowEval(nn.Module):
+    def __init__(
+        self,
+        mean: Tuple[float, ...] = (0.5, 0.5, 0.5),
+        std: Tuple[float, ...] = (0.5, 0.5, 0.5),
```
[the remark this answers is not in the document]
Comment (datumbox) [reply torchvision/transforms/_presets.py:112]: Are you sure this is the only normalization strategy that makes sense in this problem? This looks to me like the TF default strategy. Perhaps the paper was first implemented in TF?

[the remark this answers is not in the document]
Comment (NicolasHug) [reply torchvision/transforms/_presets.py:112]: The RAFT model expects input in [-1 ,1]. We don't support any other model ATM so I'm not sure it makes sense to support any other normalization. This is something we could add in a BC way later if we need to or if users ask for it?

**Review by NicolasHug — APPROVED** [review]
Thanks @datumbox , LGTM modulo the built example gallery look OK visually.
In the future we should go back through them and probably slightly edit some parts in the narration which may become outdated due to the new API (typically when we tell users they should normalize, etc.)

**Comment by datumbox** [comment]
@NicolasHug Absolutely. I'm just taking care of code changes for now and ensuring that nothing breaks. We should put some serious effort on the docs once porting is completed. Thanks for the review.
